stable/gocd/README.md

```diff
@@ -224,6 +224,28 @@ $ helm install --name gocd-app --set server.persistence.existingClaim=PVC_NAME s
 1. That packages being cached here is shared between all the agents.
 2. That all the agents sharing this directory are privy to all the secrets in `/home/go`
 
+## SSH keys
+For accessing repositories over SSH, you need to add SSH keys to the server and the agent.
+Generate a new keypair, fetch the host key for the [host] you want to connect to and create the secret.
+The secret is structured to hold the entire contents of the .ssh folder on the GoCD server and agent.
+ ```bash
+$ ssh-keygen -t rsa -b 4096 -C "user@example.com" -f gocd-ssh -P ''
+$ ssh-keyscan [host] > gocd_known_hosts
+$ kubectl create secret generic gocd-ssh \
+    --from-file=id_rsa=gocd-ssh \
+    --from-file=id_rsa.pub=gocd-ssh.pub \
+    --from-file=known_hosts=gocd_known_hosts
+```
+ The last step is to copy the key over to the host, so GoCD can connect.
+
+### SSH key configuration parameters
+ | Parameter                                     | Description                                         | Default              |
+| --------------------------------------------- | --------------------------------------------------- | -------------------- |
+| `server.security.ssh.enabled`                 | Enable the use of SSH keys for GoCD server          | `false`              |
+| `server.security.ssh.secretName`              | The name of the secret holding the SSH keys         | `gocd-ssh`           |
+| `agent.security.ssh.enabled`                  | Enable the use of SSH keys for GoCD agent           | `false`              |
+| `agent.security.ssh.secretName`               | The name of the secret holding the SSH keys         | `gocd-ssh`           |
```

Comment:
Mind moving this documentation to server and agent section separately?

Reply:
updated the read me files. Hope it looks good now. let me know if any improvement can be done.